android/app/src/main/kotlin/org/getlantern/lantern/model/LanternSessionManager.kt

```diff
@@ -17,6 +21,11 @@ class LanternSessionManager(application: Application) : SessionManager(applicati
 
     private var referral: String? = null
     private var verifyCode: String? = null
+    private val supportedCurrencyCodes = arrayOf(
+        "rub", "sar", "inr", "sgd", "jpy", "aud", "try", "mop", "cad", "vnd",
+        "thb", "hkd", "bhd", "eur", "php", "cny", "irr", "krw", "myr", "gbp",
+        "twd", "idr", "usd", "aed", "ils"
+    )
```

Comment:
Do we know how often this list of supported currency codes is updated? I am just wondering if we end up adding a new one to the list, if we would run into issues having a bunch of old clients on an old version without it. If it changes infrequently though, I think that's fine

Reply:
@atavism  I think we should just get rid of this currency codes list, whatever we get from the device, we should just send it to the server, if the currency is not supported it will throw an error on the server and the server can update currency easily, What do you think?